stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -370,18 +237,13 @@ async def update_item(
         """
         ...
 
+    @abc.abstractmethod
     async def patch_item(
         self,
         collection_id: str,
         item_id: str,
-        patch: Union[stac.PartialItem, List[stac.PatchOperation]],
-        content_type: Optional[
-            Literal[
-                "application/json-patch+json",
-                "application/merge-patch+json",
-                "application/json",
-            ]
-        ] = "application/json",
+        patch: Any,
```

Comment:
could `patch` still be of type `Union[stac.PartialItem, List[stac.PatchOperation]],` ?

Reply:
It could but other PATCH implementations like [XML PATCH](https://datatracker.ietf.org/doc/html/rfc5261) would have a different type so for future proofing I set it to `Any`. But can switch back if we think most will use JSON merge or patch. I guess this can be overwritten in the backend if other types are needed?

Reply:
🤔 the spec doesn't mention XML patch but only `RFC 7396` and `RFC 6902`. If we ever add `RFC 5261` to the spec we could update it here but for now I think it's safe to exclude it